manifest_staging/charts/csi-secrets-store-provider-azure/templates/arc-monitoring.yaml

```diff
@@ -0,0 +1,247 @@
+        # Pipeline agent for logging
+        - name: amacoreagent
+          image: "pipelineagent.azurecr.io/amacoreagent:3.0"
+          imagePullPolicy: IfNotPresent
+          resources:
+            requests:
+              cpu: 50m
+              memory: 100Mi
+            limits:
+              cpu: 50m
+              memory: 100Mi
+          env:
+          - name: MONITORING_GCS_AUTH_ID_TYPE
+            value: AuthMSIToken
+          - name: PA_TENANTS_DIR
+            value: /etc/tenants
+          - name: PA_GIG_BRIDGE_MODE
+            value: "1"
+          - name: PA_FLUENT_SOCKET_PORT
+            value: "15000"
+          - name: PA_TRACEFLAGS
+            value: "4"
+          - name: OTLP_DEBUG_VERBOSE
+            value: "1"
+          - name: FLUENTD_PORT
+            value: "8130"
+          - name: MDSD_DEBUG_LOG_FLAGS
+            value: "-T 0x2002"
+          - name: TENANT
+            value: "westus2"
+          - name: ROLE
+            value: "akvsecretsprovider"
+          - name: ROLEINSTANCE
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
+          - name: MONITORING_TENANT
+            value: "westus2"
+          - name: MONITORING_ROLE
+            value: "akvsecretsprovider"
+          - name: MONITORING_ROLE_INSTANCE
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
+          - name: MONITORING_GCS_ENVIRONMENT
+            value: "DiagnosticsPROD"
+          - name: MONITORING_GCS_ACCOUNT
+            value: "akvsecretsprovider"
+          - name: MONITORING_GCS_REGION
+            value: "westus2"
+          - name: MONITORING_GCS_NAMESPACE
+            value: "akvsecretsprovider"
+          - name: MONITORING_CONFIG_VERSION
+            value: "1.0"
+          - name: MDSD_CONFIG_DIR
+            value: /tmp
+          - name: DOCKER_LOGGING
+            value: "true"
+          volumeMounts:
+            - name: tenants-conf-vol
+              mountPath: /etc/tenants
+            - name: var-log-vol
+              mountPath: /geneva/geneva_logs
+            - name: mdsd-run-vol
+              mountPath: /var/run/mdsd
+        # FluentD
+        - name: fluentd
+          image: "linuxgeneva-microsoft.azurecr.io/genevafluentd_td-agent:master_20220403.1"
+          imagePullPolicy: IfNotPresent
+          resources:
+            requests:
+              cpu: 50m
+              memory: 100Mi
+            limits:
+              cpu: 50m
+              memory: 100Mi
+          env:
+            - name: FLUENTD_CONF
+              value: /etc/fluentd/fluentd.conf
+          volumeMounts:
+            - name: fluentd-conf-vol
+              mountPath: /etc/fluentd
+            - name: fluentd-buffer-vol
+              mountPath: "/var/log/td-agent"
+            - name: mdsd-run-vol
+              mountPath: "/var/run/mdsd"
+            - name: docker-log-vol
+              mountPath: /var/lib/docker/containers
```

Comment:
Is this relevant if the cluster is running containerd?

Reply:
nope.